.github/minty.yaml

```diff
@@ -2,7 +2,7 @@ version: 'minty.abcxyz.dev/v2'
 
 rule:
   if: |-
-    assertion.iss == 'https://token.actions.githubusercontent.com' &&
+    assertion.iss == issuers.github &&
```

Comment:
@drevell is it safe to change this now or do I need to wait for minty to deploy before updating its own config?

Reply:
Better ask Brad.

Reply:
you will need to redeploy.